docs/site/src/pages/reference/concepts/channels.mdx

```diff
@@ -14,86 +18,102 @@ visit the respective pages for the [Python](../python-client/channels) and
 
 <Divider.Divider direction="x" />
 
-## What is a Channel?
+## Introduction
 
 A channel is a logical collection of samples emitted by or representing the
```

Comment:
can we just say "A channel is a logical grouping of samples that represent the values from a single source". not sure the emitted by or representing part really adds more clarity

Reply:
Yes. Corrected